jerry-core/lit/lit-char-helpers.c

```diff
@@ -63,13 +121,21 @@ lit_char_is_space_separator (ecma_char_t c) /**< code unit */
 bool
 lit_char_is_white_space (ecma_char_t c) /**< code unit */
 {
-  return (c == LIT_CHAR_TAB
-          || c == LIT_CHAR_VTAB
-          || c == LIT_CHAR_FF
-          || c == LIT_CHAR_SP
-          || c == LIT_CHAR_NBSP
-          || c == LIT_CHAR_BOM
-          || lit_char_is_space_separator (c));
+  if (c <= 127)
+  {
+    return c == LIT_CHAR_TAB
+    || c == LIT_CHAR_VTAB
+    || c == LIT_CHAR_FF
+    || c == LIT_CHAR_SP;
```

Comment:
If I indent these lines I get error messages like this when running precommit:
> lit-char-helpers.c:127: error: Indentation: 12 -> 4. Line: '            || c == LIT_CHAR_VTAB'
>lit-char-helpers.c:128: error: Indentation: 12 -> 4. Line: '            || c == LIT_CHAR_FF'
>lit-char-helpers.c:129: error: Indentation: 12 -> 4. Line: '            || c == LIT_CHAR_SP;'
What could be the problem?